Posts/2021/11/tfl-formatenumeration.md

```diff
@@ -0,0 +1,150 @@
+---
+post_title: How to Use $FormatEnumerationLimit
+username: tfl@psp.co.uk
+Categories: PowerShell
+tags: PowerShell, Format, FormatEnumerationLimit variagble
+Summary: Using The $FormatEnumerationLimit variable in PowerShell
+---
+
+**Q:** When I format an object where a property contains more than 4 objects, I never see the extra property values. How can I fix that?
+
+**A:** Use the `$FormatEnumerationLimit` variable.
+
+This query is one I hear in many PowerShell support forums, and I have encountered this issue a lot over the years.
+What happens is that you issue a command to return objects.
+PowerShell returns the requested objects with properties that are arrays of objects, not a single property value.
```

Comment:
This sentence doesn't really make sense. It sounds like that the properties are arrays of objects. That can be true but the cmdlet also returns arrays of objects. So it is not clear what you mean here.

```suggestion
The objects returned by a `Get-*` cmdlet can contain properties that are arrays of values, not just a single value.
```

Reply:
I've reworded your suggestion.

To be pedantic, some properties are going to be arrays of objects. The modules  property of the Process object for example. That property is a full blown array of objects. :-)